pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -153,6 +154,7 @@ func (r *ReconcileCincinnati) Reconcile(request reconcile.Request) (reconcile.Re
 		r.ensureGraphBuilderService,
 		r.ensurePolicyEngineService,
 		r.ensurePodDisruptionBudget,
+		r.ensurePolicyEngineRoute,
```

Comment:
I only created route for the policy-engine, which exposes an HTTP API. It seems to me the graph-builder doesn't need to be exposed?

Reply:
My understanding is graph-builder doesn't need to be exposed.